packages/renderer/src/lib/preferences/PreferencesKubernetesContextsRendering.svelte

```diff
@@ -243,14 +257,25 @@ async function connect(contextName: string): Promise<void> {
         <div class="grow flex-column divide-gray-900 text-[var(--pd-invert-content-card-text)]">
           <div class="flex flex-row">
             <div class="flex-none w-36">
-              {#if context.isReachable}
+              {#if context.isReachable || context.isOffline}
                 <div class="flex flex-row pt-2">
-                  <div class="w-3 h-3 rounded-full bg-[var(--pd-status-connected)]"></div>
-                  <div
-                    class="ml-1 font-bold text-[9px] text-[var(--pd-status-connected)]"
-                    aria-label="Context Reachable">
-                    REACHABLE
-                  </div>
+                  {#if context.isOffline}
+                    <Tooltip class="flex flex-row" tip="connection to the cluster lost, the resources displayed can be out of date">
```

Comment:
suggestion: The message is a bit long (and cluster/displayed are implied), 'may be' is more natural than 'can be', and maybe 'out of sync' is more common?

'connection lost, resources may be out of sync'?